### PR TITLE
docs: describe private properties

### PR DESCRIPTION
**Pull request by jaw0r3k**
**Please describe the changes this PR makes and why it should be merged:**

**Status and versioning classification:**

- There are no code changes
- I know how to update typings and have done so
- This PR changes the library's interface (methods or parameters added)
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: packages/discord.js/src/managers/CachedManager.js [hunk 1]
```diff
@@ -11,6 +11,13 @@ class CachedManager extends DataManager {
   constructor(client, holds, iterable) {
     super(client, holds);
 
+    /**
+     * The private cache of items for this manager.
```
Comment (MrMythicalYT) [inline packages/discord.js/src/managers/CachedManager.js:15]:
```suggestion
     * The private cache of items for this manager
```

File: packages/discord.js/src/structures/GuildMember.js [hunk 2]
```diff
@@ -53,6 +53,11 @@ class GuildMember extends Base {
      */
     this.communicationDisabledUntilTimestamp = null;
 
+    /**
+     * The role ids of this member
```
Comment (MrMythicalYT) [inline packages/discord.js/src/structures/GuildMember.js:57]:
```suggestion
     * The role ids of the member
```
Could probably be worded more differently

File: packages/discord.js/src/structures/Presence.js [hunk 3]
```diff
@@ -285,6 +292,13 @@ class Activity {
  */
 class RichPresenceAssets {
   constructor(activity, assets) {
+    /**
+     * The activity of the RichPresenceAssets
+     * @type {Activity}
+     * @private
```
Comment (MrMythicalYT) [inline packages/discord.js/src/structures/Presence.js:298]:
```suggestion
```
Not sure if this should be private

File: packages/discord.js/src/structures/Presence.js [hunk 4]
```diff
@@ -141,6 +141,13 @@ class Presence extends Base {
  */
 class Activity {
   constructor(presence, data) {
+    /**
+     * The presence of the Activity
+     * @type {Presence}
+     * @private
```
Comment (MrMythicalYT) [inline packages/discord.js/src/structures/Presence.js:147]:
```suggestion
```
Not sure if this should be private

File: packages/discord.js/typings/index.d.ts [hunk 5]
```diff
@@ -227,6 +227,7 @@ declare module 'node:events' {
 
 export class Activity {
   private constructor(presence: Presence, data?: RawActivityData);
+  private readonly presence: Presence;
```
Comment (MrMythicalYT) [inline packages/discord.js/typings/index.d.ts:230]: Not sure if this should be private

File: packages/discord.js/typings/index.d.ts [hunk 6]
```diff
@@ -2348,6 +2350,7 @@ export class ReactionEmoji extends Emoji {
 
 export class RichPresenceAssets {
   private constructor(activity: Activity, assets: RawRichPresenceAssets);
+  private readonly activity: Activity;
```
Comment (MrMythicalYT) [inline packages/discord.js/typings/index.d.ts:2353]: Not sure if this should be private

**Review by MrMythicalYT — CHANGES_REQUESTED** [review]

File: packages/discord.js/src/managers/CachedManager.js (hunk 1 above)
Comment (jaw0r3k) [reply packages/discord.js/src/managers/CachedManager.js:15]: why without `.`

Comment (MrMythicalYT) [reply packages/discord.js/src/managers/CachedManager.js:15]: It's hard to tell the correct format IMO. One property has a `.` in description, one doesn't 😅

File: packages/discord.js/src/structures/Presence.js (hunk 3 above)
Comment (jaw0r3k) [reply packages/discord.js/src/structures/Presence.js:298]: Also not sure but cool that they will be accesable to users

File: packages/discord.js/typings/index.d.ts (hunk 6 above)
Comment (jaw0r3k) [reply packages/discord.js/typings/index.d.ts:2353]:
````suggestion
  public readonly activity: Activity;
```

File: packages/discord.js/typings/index.d.ts (hunk 5 above)
Comment (jaw0r3k) [reply packages/discord.js/typings/index.d.ts:230]:
```suggestion
  public readonly presence: Presence;

**Review by vladfrangu — APPROVED** [review]
Sure

**Review by SpaceEEC — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Pull request merged**